app/cli/cmd/auth_login.go

```diff
@@ -71,6 +71,7 @@ func interactiveAuth() error {
 	// Append local callback URL
 	q := serverLoginURL.Query()
 	q.Set("callback", callbackURL.String())
+	q.Set("long-lived", "true")
```

Comment:
note: This change will require users to update their CLIs otherwise their tokens will expire constantly.

I decided to go this way because

- It just affects operators CLI, not the attestation CLI
- Doing the opposite, meaning making long-lived default while short-lived optional is not good practice.
- We are still pre-1.0 so we do not guarantee backwards compatibility.
- I do not think we can detect the version of the CLI yet, so there is no way to show a warning message to the user.

Please let me know if you have any objection on this breaking change.

Reply:
Can you please elaborate on "It just affects operators CLI, not the attestation CLI"?
You mean that it does not affect RobotAccount auth based calls?

Reply:
Correct